### PR TITLE
[4.0] Libraries src part2

### PR DESCRIPTION
**Pull request by brianteeman**
Continuing the namespace work

NOTE there are a lot of files in this folder and its hard to separate into smaller pr so I have decided to do it in batches ie test and merge this one which is the main namespacing and then do subsequent pr once merged for the remaining namespaces

### CODE REVIEW

**Comment by brianteeman** [comment]
Pretty sure that the drone errors are for a system test that is unrelated to this - please advise

File: libraries/src/Component/Router/Rules/MenuRules.php [hunk 1]
```diff
@@ -162,7 +163,7 @@ public function preprocess(&$query)
 
 		// Check if the active menuitem matches the requested language
 		if ($active && $active->component === 'com_' . $this->router->getName()
-			&& ($language === '*' || in_array($active->language, array('*', $language)) || !\JLanguageMultilang::isEnabled()))
+			&& ($language === '*' || in_array($active->language, array('*', $language)) || !\LanguageMultilang::isEnabled()))
```
Comment (Quy) [inline libraries/src/Component/Router/Rules/MenuRules.php]: Remove `\`

**Comment by wilsonge** [comment]
Thanks guys!

**Comment by brianteeman** [comment]
Thanks - now to do part 3 :(

**Pull request merged**